ops/dev5/_data.tf

```diff
@@ -235,3 +235,8 @@ data "azurerm_key_vault_secret" "datahub_signing_key" {
   name         = "datahub-signing-key-test"
   key_vault_id = data.azurerm_key_vault.sr_global.id
 }
+
+data "azurerm_key_vault_secret" "slack_hook_token" {
```

Comment:
should we also update this for the other lowers as well or will you be doing that in a separate PR?

Reply:
yeah maybe something worth getting other folks to chime in on, but I didn't want to add anything to the lower envs since there really shouldn't be a reason (other verifying things work as in this PR) for those secrets to be there. Wouldn't want a situation where we get a false positive from testing there. Plus, folks can always plug in the token locally if they need to configure something.

If folks agree, I'll probably go back and delete this token on dev5 after this goes in

Reply:
This reasoning makes sense to me! Thank you, Bob!